incl/misc/getTopArtists.php

```diff
@@ -26,35 +27,41 @@
 	$robsult = curl_exec($ch);
 	curl_close($ch);
 	echo $robsult;
-} else {
+}
+if($redirect == 0) {
 	// select
 	$querywhat = "SELECT authorName, download FROM songs WHERE (authorName NOT LIKE '%Reupload%' AND authorName NOT LIKE 'unknown') GROUP BY authorName ORDER BY COUNT(authorName) DESC LIMIT 20 OFFSET $offset"; // offset couldn't be used in prepare statement for some very odd reason
 	$query = $db->prepare($querywhat);
 	$query->execute();
 	$res = $query->fetchAll();
 	// count
-	$countquery = $db->prepare("SELECT count(DISTINCT(authorName)) FROM songs WHERE (authorName NOT LIKE '%Reupload%' AND authorName NOT LIKE 'unknown')");
-	$countquery->execute();
-	$totalCount = $countquery->fetchColumn();
+	$countquery = $res;
+	$totalCount = 100;
+	// $countquery = $db->prepare("SELECT count(DISTINCT(authorName)) FROM songs WHERE (authorName NOT LIKE '%Reupload%' AND authorName NOT LIKE 'unknown') ORDER BY COUNT(authorName) DESC LIMIT 20 OFFSET $offset");
+	// $countquery->execute();
+	// $totalCount = $countquery->fetchColumn();
 	// parse
 	foreach($res as $sel){
-		$str .= "4:$sel[0]";
+		$str .= "4:".$sel[0];
 		// TO-DO: Fetch YouTube links from RobTop's servers, as we are unable to auto-determine YouTube links.
 		// Also credit to @Intelligent-Cat for this piece of code
-		if (substr($sel[1], 0, 26) == "https://api.soundcloud.com") {
+
+		if (substr($sel[1], 0, 26) != "https://api.soundcloud.com") {
```

Comment:
...this was specifically made *for* Links that could redirect to soundcloud so I don't see why you replaced this with !=